tap_typeform/streams.py

```diff
@@ -292,76 +318,52 @@ def sync_forms(atx):
             continue
 
         # start_date is defaulted in the config file 2018-01-01
-        # if there's no default date and it gets set to now, then start_date will have to be
-        #   set to the prior business day/hour before we can use it.
-
+        # if there's no default date and it gets set to now
         now = datetime.datetime.now(pytz.utc)
-        if incremental_range == "daily":
-            s_d = now.replace(hour=0, minute=0, second=0, microsecond=0)
-            start_date = pendulum.parse(atx.config.get('start_date', s_d + datetime.timedelta(days=-1, hours=0)))
-        elif incremental_range == "hourly":
-            s_d = now.replace(minute=0, second=0, microsecond=0)
-            start_date = pendulum.parse(atx.config.get('start_date', s_d + datetime.timedelta(days=0, hours=-1)))
-        LOGGER.info('start_date: {} '.format(start_date))
-
-
-        # end date is not usually specified in the config file by default so end_date is now.
-        # if end date is now, we will have to truncate them
-        # to the nearest day/hour before we can use it.
-        if incremental_range == "daily":
-            e_d = now.replace(hour=0, minute=0, second=0, microsecond=0).strftime("%Y-%m-%d %H:%M:%S")
-            end_date = pendulum.parse(atx.config.get('end_date', e_d))
-        elif incremental_range == "hourly":
-            e_d = now.replace(minute=0, second=0, microsecond=0).strftime("%Y-%m-%d %H:%M:%S")
-            end_date = pendulum.parse(atx.config.get('end_date', e_d))
-        LOGGER.info('end_date: {} '.format(end_date))
+        s_d = now.replace(hour=0, minute=0, second=0, microsecond=0)
+        start_date = pendulum.parse(atx.config.get('start_date', s_d))
 
         # if the state file has a date_to_resume, we use it as it is.
         # if it doesn't exist, we overwrite by start date
         s_d = start_date.strftime("%Y-%m-%d %H:%M:%S")
         last_date = pendulum.parse(get_bookmark_value(atx.state, stream_name, form_id, default=s_d))
```

Comment:
Again, this default value seems wrong. I think `parse()` would throw an error

Reply:
This should be fine as `s_d` is being overwritten with the `strftime` method which returns a string

```Python
>>> import pendulum
>>> start_date = pendulum.parse("2021-09-22T12:34:56Z")
>>> start_date
DateTime(2021, 9, 22, 12, 34, 56, tzinfo=Timezone('UTC'))
>>> s_d = start_date.strftime("%Y-%m-%d %H:%M:%S")
>>> s_d
'2021-09-22 12:34:56'
>>> pendulum.parse(s_d)
DateTime(2021, 9, 22, 12, 34, 56, tzinfo=Timezone('UTC'))
```

Reply:
You're right.
I saw
```python
s_d = now.replace(hour=0, minute=0, second=0, microsecond=0)
```
and not
```python
s_d = start_date.strftime("%Y-%m-%d %H:%M:%S")
```